rssalertbot/feed.py

```diff
@@ -127,11 +127,12 @@ async def _fetch(self, session, timeout=10):
                     return await response.text()
 
             except concurrent.futures.CancelledError:
-                self._handle_fetch_failure('Timeout', f"Timeout while fetching feed")
+                self._handle_fetch_failure('Timeout', "Timeout while fetching feed")
 
             except Exception as e:
                 etype = '.'.join((type(e).__module__, type(e).__name__))
-                self._handle_fetch_failure('Exception', f"{etype} fetching feed: {e}")
+                await self._handle_fetch_failure('Exception', f"{etype} fetching feed: {e}")
+                self.log.exception("Error fetching feed")
```

Comment:
I think it's fine to log this as `error` via `_handle_fetch_failure` in order to retain the alerting functionality.

Reply:
I specifically wanted the exception trace itself to help diagnose this, and wanted to do it outside of another async call. Though it now occurs to me I should swap the order of lines 134 and 135 to log first.

Reply:
In that case, the two other calls to `_handle_fetch_failure` should be updated to log independently of the method call.